configs/myconfigs/sample.py

```diff
@@ -289,6 +302,7 @@ def handler(signum, frame):
 
         if exit_event.getCause() == "work started count reach":
             simulateROI(system, roiInstructions, samples)
+            break       #  exit after ROI is complete
```

Comment:
The way I see it, we might require to break if we are not sampling the entire ROI. In this case it does not make much difference to the wall clock time of this script. But in simulate_one.py the run time of all sample points may get delayed waiting for the end of the application. Either ways is fine here..